src/app/card/card.component.ts

```diff
@@ -27,15 +27,15 @@ value: any;
       }
 
       });
-      dialogRef.afterClosed().subscribe((t:any)=>{})
+      dialogRef.afterClosed().subscribe()
         
   
 
   }
   delete(id:any) {
       
-    this.service.deleteOrder(id).subscribe(d=>
-      {this.router.navigate(["dashboard"])});
+    this.service.deleteOrder(id).subscribe()
+      {this.router.navigate(["dashboard"])};
```

Comment:
this is not the right way, router navigate should come inside subscribe

Reply:
resolved